src/assets/styles/loading.css

```diff
@@ -1,276 +0,0 @@
-.empty-space::before {
```

Comment:
Funny - I don't see this `empty-space` class anywhere.  So I guess this file was doing nothing?